be/test/vec/columns/column_array_test.cpp

```diff
@@ -0,0 +1,425 @@
+            InputTypeSet input_types {};
+            input_types.push_back(array_typeIndex[i][0]);
+            input_types.push_back(Nullable {static_cast<TypeIndex>(array_typeIndex[i][1])});
+            EXPECT_EQ(input_types[1].type(), &typeid(Nullable)) << "nested type is not nullable";
+            EXPECT_TRUE(parse_ut_data_type(input_types, descs[i]));
+        }
+
+        // create column_array for each data type
+        vector<string> data_files = {data_file_dir + "test_array_tinyint.csv",
+                                     data_file_dir + "test_array_smallint.csv",
+                                     data_file_dir + "test_array_int.csv",
+                                     data_file_dir + "test_array_bigint.csv",
+                                     data_file_dir + "test_array_largeint.csv",
+                                     data_file_dir + "test_array_float.csv",
+                                     data_file_dir + "test_array_double.csv",
+                                     data_file_dir + "test_array_ipv4.csv",
+                                     data_file_dir + "test_array_ipv6.csv",
+                                     data_file_dir + "test_array_date.csv",
+                                     data_file_dir + "test_array_datetime.csv",
+                                     data_file_dir + "test_array_date.csv",
+                                     data_file_dir + "test_array_datetimev2(6).csv",
+                                     data_file_dir + "test_array_varchar(65535).csv",
+                                     data_file_dir + "test_array_decimalv3(7,4).csv",
+                                     data_file_dir + "test_array_decimalv3(16,10).csv",
+                                     data_file_dir + "test_array_decimalv3(38,30).csv",
+                                     data_file_dir + "test_array_decimalv3(76,56).csv"};
+
+        // step2. according to the datatype to make column_array
+        //          && load data from csv file into column_array
+        EXPECT_EQ(descs.size(), data_files.size());
+        for (int i = 0; i < array_typeIndex.size(); i++) {
+            auto& desc = descs[i];
+            auto& data_file = data_files[i];
+            // first is array type
+            auto& type = desc[0].data_type;
+            std::cout << "type: " << type->get_name() << " with file: " << data_file << std::endl;
+            MutableColumns columns;
+            columns.push_back(type->create_column());
+            auto serde = type->get_serde(1);
+            load_data_from_csv({serde}, columns, data_file, ';');
+            array_columns.push_back(std::move(columns[0]));
+            array_types.push_back(type);
+            serdes.push_back(serde);
+        }
+        // step3. show array column data
+        for (int i = 0; i < array_columns.size(); i++) {
+            //            auto& column = array_columns[i];
+            //            printColumn(*column, *descs[i][0].data_type);
```

Comment:
Do you mean some check on data?

Reply:
yes this is just for me to debug data is right or not